stdlib/public/core/CTypes.swift

```diff
@@ -220,6 +241,7 @@ extension CVaListPointer : CustomDebugStringConvertible {
     return value.debugDescription
   }
 }
+#endif
```

Comment:
This `#endif` should be up a few lines. The `CustomDebugStringConvertible` should be valid for all definitions.

Reply:
Naw, its not.  The value type is different in all cases.  In fact, for the AAPCS64 case, its a tuple, rather than an `UnsafeMutablePointer`.

Reply:
Saleem and I briefly talked about this at the LLVM dev meeting, and I told him to go ahead without it. I'm not sure we have a good debug description for a struct-based va_list that's any better than just showing the children (the default behavior of `String(reflecting:)`).